### ISSUE TITLE
Allow categories sorting

### ISSUE THREAD

**Issue by victor-yunenko** [nephila/djangocms-blog#615]
This is quite easy to do, I can implement that until Wednesday.

![image](https://user-images.githubusercontent.com/36416236/86476759-d0b7e480-bd4f-11ea-8b48-430556314ecf.png)

**Comment by yakky** [comment]
It can be a nice addition. It must be implemented in a backward compatible way and it should not force users to arrange categories manually if they want to keep an alphabetical order

**Comment by victor-yunenko** [comment]
Makes sense to me - I'll set name as the default sorting after the `order` field 👍

**Comment by yakky** [comment]
:+1:
